bench/perf_run_py/perf_stat_collection.py

```diff
@@ -112,14 +108,10 @@ def all_stats():
                             output_label='Latency(read, update) Max',
                             ops=['read', 'update'],
                             num_max=1),
-            PerfStatMax(short_label="max_read_throughput",
-                        pattern=r'updates,',
-                        input_offset=4,
-                        output_label='Max read throughput'),
-            PerfStatMin(short_label="min_read_throughput",
-                        pattern=r'updates,',
-                        input_offset=4,
-                        output_label='Min read throughput'),
+            PerfStatMinMax(short_label="min_max_read_throughput",
+                           pattern=r'updates,',
+                           input_offset=4,
+                           output_label='Read throughput'),
```

Comment:
I think that since the `output_label` is changed we won't see previous (min, max) graphs in evergreen and Atlas (after this merge). 
Basically, if we change the `test name`, `output_label` or `task name`, we end up breaking graphs. 
[here](https://github.com/wiredtiger/wiredtiger/pull/7246/files#diff-7f1db082bab724a4789aac58cec9f4e191ef593c2a46803970b0a845acb5f66fR94) we can change it to `Min {self.output_label}`?

Reply:
Yep, no worries. 

ca4cb99 changes this to maintain the prior `output_labels`:
```
"metrics": [
        {
            "name": "Min update throughput",
            "value": 3120286
        },
        {
            "name": "Max update throughput",
            "value": 3120286
        },
        {
            "name": "All update throughputs",
            "values": [
                3120286,
                ...
            ]
        }
```